### PR TITLE
Add the Feature to add to gyms?

### PR DESCRIPTION
**Pull request by BuntyChops**
Would it be possible to add the feature to add to a nearby gym that is either already your team, or an open gym?

Say add your highest CP from your favourites so you can set which ones you will add to gyms?

### CODE REVIEW

**Comment by PassiveModding** [comment]
I dont think this would be a good idea, gyms are all that legit players have left and this would probably ruin it for them.

At most only have the option to add a certain pokemon to a gym of your team with an empty space, no more

**Comment by littlemonster1** [comment]
but even then, depending on the timing, you might take the spot away from a legit player if the team just took over the gym and you add a pokemon before they can. Maybe even after the spend a lot of time training it ...

**Comment by globus243** [comment]
if you automate the only game mechanic that gives you free coins, niantic will shoot down this project faster then you could implement it because that's how they earn money and pokemobbot will hurt there cash flow

**Comment by mossyblog** [comment]
Ummm there's really no DMCA implication being invoked here so "shutting it down" has no legal merit, nor has it technically impacted the EULA given it's encumbered upon the existence of a compiled source by an end user who invokes that legality (ie you compile and run it, you breach EULA...)

Lastly, everything about this project impact their revenue models, so if they could, they would - regardless of Fort.StartGymAttack being finished.

**Comment by globus243** [comment]
Farming Stops and Pokes is not the same as farming coins you normally pay for.
I mean it could happen nothing, but do you really want take chances.

**Comment by MartyMcPidgey** [comment]
Even though i'd love a bot to beat gyms for me, its true. it will ruin everything for physical players, and after time, make it worthless for us aswel. (whats the fun if you already own every gym?)

On the other side.. How long will it take before another bot does release this feature just to get more people paying/supporting it.

I don't have a strong opinion on it. For now, I'd say; let the gyms be what they are.

**Comment by nyirsh** [comment]
Good point there. As a dev I have interests into making a similar functionality because of "challenge purposes" but I would never ever make it public or use actively use it even if it works not only for the reasons stated above but also for this other one: how much time it will take before another bot takes out the gym you just took?

**Comment by MartyMcPidgey** [comment]
@nyirsh didn't even think of that. Now you mention it, imagine the gyms at central park. I wonder how many virtual players are walking around there. If someone would automate that, it would be wise to cash in those 10 coins as soon as you got a gym. With 3 teams, it would be too hard keeping one.

EDIT: 

Additional thoughts: After this would be released, one of the most popular tools would become a gym tracker. Where you can see what team a gym belongs to. Imagine a "pokevision" kind of app just displaying gyms with their prestige. going up and down...

**Comment by CoderFX** [comment]
@PassiveModding
By saying "legit players" you mean rich schoolkids who buy tons of worthless 1'ns and 0'ros to speed their leveling?

**Comment by varunchopra** [comment]
@CoderFX I don't know where the fuck you're from but, where I'm from:
a, school kids are _not_ rich, their parents are.
and b, not every legit player is a child.

All of us have a moral obligation to not ruin the game we've waited so many years for. Then again, do what you like. I hope the ban waves start coming soon.

**Comment by MartyMcPidgey** [comment]
@CoderFX If you take out the fun of a game, no players will get back. Result of that; the game will die soon. And with a dead game, there is no fun of botting. It's a spiral. Keep the fun in it for players will keep the fun for us. Not just school kids.

If you ever played runescape in the good old days, bots and players kinda had an ongoing affair. Bots got the resources players sold. Noone really talked about it, and not many people botted. It was fine. We all had resources that were boring to get, and if you got the money, you'd be fine. Everyone was happy. untill botting was made too easy. everybody got their own resources PLUS they wanted to sell everything. Even though there were mass ban waves, almost everyone quit. (plus they fucked up a perfectly fine game, but thats another story.)

I myself have my botting account and a legit account. Know the difference.

As @chopraaa said, not every legit player is a child. many friends of it play it. Together with me. And i  enjoy it. We've been waiting for this game, and I'm really hoping this game will evolve. More options, more generations, maybe even a storyline. Give us what we all always wanted. Catch them all. And not let it be too easy.

TL;DR: Without actual playing, botting cannot exist. 
"no shadow without light"-kinda stuff.

**Comment by marksterxxx** [comment]
and this pull request is for?

**Comment by LNSDISCORDER** [comment]
wow please remove this pull request, i don't believe its the place to debate it. However, that wont stop me now its here. 

I am super afraid people still think this is a good idea. Its not, once you mess with gyms we will find it harder and harder to bot. 

My opinion? No need to be greedy and bot the entire game, especially when you will really upset legit players, and when you do that, ni-antic will take action to secure their income stream. At the moment we are not messing with their income, so why start. Its stupid, greedy, going to far and its reckless and not thinking about the longevity of botting pokego..

@mossyblog what you said is narrow minded and completely ridiculous. Pokego botting in its current state actually creates competition for the legit players without noticeably effecting their ability to take gyms. They will spend money to catch up ect ect, (Thinking they are behind other legit players thus motivating them to spend $ to help catch up) 

If we take gyms then they will get upset, very upset and it will be noticeable and a easy search will provide them ammo to use as there will be "gym bot" plastered everywhere. 

Please reconsider your logic. Most companies don't unofficially mind bots because they do actually create content and competition to legit players that motivate them to spend. 

Taking the gyms will be too much, it will be going too far, it will be too noticeable. Saying "we are doing it anyhow so why not do it all the way" is a sick way to look and justify things, yet it happens all too often by narrow minded individuals that dont have any experience in the area. People that only factor in their narrow bias. 

I encourage you to purge some of that bullshit from your logical common sense process that seems broken to me.

@CoderFX "By saying "legit players" you mean rich schoolkids who buy tons of worthless 1'ns and 0'ros to speed their leveling?"

Its not worthless if people spend a pegged currency to purchase it. Are we learning economics today? What you posted was the ideal example of what someone does when they have nothing to contribute or imagination past a jerking off experience. Even then im sure its very much below standard.

**Comment by Jazed** [comment]
We don't plan on adding gym's as it ruins the game for legitimate players. Your pull request also conflicts.

**Comment by l3uddz** [comment]
My fork already has this if you really want to implement it when they fix the api.
